labgrid/remote/client.py

```diff
@@ -727,13 +727,15 @@ def power(self):
         target = self._get_target(place)
         from ..resource.power import NetworkPowerPort, PDUDaemonPort
         from ..resource.remote import NetworkUSBPowerPort, NetworkSiSPMPowerPort
-        from ..resource import TasmotaPowerPort
+        from ..resource import TasmotaPowerPort, YKUSHPowerPort
```

Comment:
```suggestion
        from ..resource import TasmotaPowerPort, NetworkYKUSHPowerPort
```
Also add the import the the resource module.